apps/sms/style/attachment.css

```diff
@@ -164,6 +185,15 @@
   background-position: left -30rem;
 }
 
+.ref-placeholder {
```

Comment:
yup, but probably we would need to open a follow up for vcal and maybe refactor the CSS rules afterwards. For the moment I'm going to use 'vcard' as spefic type in the CSS rules